src/lf_mask.rs

```diff
@@ -14,6 +14,7 @@ use crate::src::levels::SegmentId;
 use crate::src::levels::TxfmSize;
 use crate::src::relaxed_atomic::RelaxedAtomic;
 use crate::src::tables::dav1d_txfm_dimensions;
+use bytemuck;
```

Comment:
We're already using `zerocopy`, so if you can do these operations with `zerocopy` instead of `bytemuck` that would be much preferred.

Reply:
Let me see if zerocopy can do the same thing. If not it's also a "relatively safe" unsafe `mem::transmute` block.